examples/postgresql/basic/basic.tf

```diff
@@ -1,33 +0,0 @@
-terraform {
```

Comment:
Please keep it for now, its still used in `templates/resources/database.md.tmpl`. We will remove it when we drop the v1 resource.